### ISSUE TITLE
Localization since the Laravel 5 update

### ISSUE THREAD

**Issue by mpopke** [DallasMuseumArt/OctoberFriends#6]
Have you guys tried overriding the strings for Friends' lang.php since upgrading to the new Laravel? I've moved the file (dma/friends/lang.php) from /app/lang to /lang as the October documentation proscribes, but it's not working anymore. I've tried putting the lang directory back under /app and into a host of other places just basically shotgun debugging it, and nothing seems to work. Google/Stack/forums have turned up nothing helpful either.

Have you gotten it to work? I'm trying to determine if I've put this in the wrong place or if something is messed up on our October install or if I've actually found a bug.

**Comment by karnold** [comment]
Everything works fine for us, however we are not doing any overrides of the
language file either.

according to the laravel docs it looks like the lang.php override should be
in resources/lang/packages/{locale}/{package} however im not sure how that
relates to the plugin namespace in october unless they use the same format.

http://laravel.com/docs/5.0/localization#overriding-package-language-files

**Comment by mpopke** [comment]
Yeah, October moves the resources directory (and subdirectories) around. Supposedly, the root directory of the October install is the default resources directory while /storage/app/ holds things like upload and media now. It's weird because I've found a couple other people having the same problem (with different plugins), but most people report that it just works (/lang) and no one has given any resolution for those people who report problems. I've commented on an open issue with the October devs, but I'm not hopeful for a speedy outcome.

**Comment by karnold** [comment]
can you link to that issue? it might be something I can fix

—
Kristen

**Comment by mpopke** [comment]
https://github.com/octobercms/october/issues/1059

**Comment by karnold** [comment]
according to the octobercms issue, this has since been resolved in dev.  Can you let me know if that fixed the issue for you?
